src/libraries/System.IO.Compression/src/System/IO/Compression/DeflateZLib/DeflateStream.cs

```diff
@@ -45,6 +46,25 @@ public DeflateStream(Stream stream, CompressionLevel compressionLevel, bool leav
         {
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="DeflateStream"/> class by using the specified stream, compression options, and whether to leave the <paramref name="stream"/> open.
```

Comment:
For consistency with the other pre-existing summaries:
```suggestion
        /// Initializes a new instance of the <see cref="DeflateStream"/> class by using the specified stream, compression options, and optionally leaves the stream open.
```